packages/clay-dropdown/src/ClayDropdownItem.soy

```diff
@@ -186,8 +190,12 @@
 			{/if}
 		"
 
-		data-onclick="{$_handleItemClick}"
+		{if not $disabled}
+			data-onchange="{$_handleItemClick}"
+		{/if}
+
 		data-onkeydown="{$_handleItemKeyDown}"
```

Comment:
Shouldn't `onKeyDown` receive the same treatment?

Reply:
Even if we move to the condition it has no effect, we have to do the `disabled` treatment inside the function to ignore the tab and find a simpler way to know this information as well.